tests/openvino/tools/calibrate.py

```diff
@@ -349,7 +349,7 @@ def map_ignored_scope(ignored):
             if op.get("attributes") is not None:
                 raise ValueError('"attributes" in the ignored operations ' "are not supported")
             ignored_operations.append(op["type"])
-    return {"ignored_scope": IgnoredScope(names=ignored.get("scope"), types=ignored_operations)}
+    return {"ignored_scope": IgnoredScope(names=ignored.get("scope", []), types=ignored_operations)}
```

Comment:
Why is this a mistake? It looks like NNCF should support empty list in `IgnoredScope`.

Reply:
> It looks like NNCF should support empty list in IgnoredScope.

I agree, so the default value was changed accordingly from None to an empty list.